rero_ils/modules/ext.py

```diff
@@ -71,6 +74,17 @@
 from ..version import __version__
 
 
+@identity_loaded.connect
+def on_identity_loaded(_, identity):
+    """Add ``Needs`` to manage RERO-ILS permissions."""
+    if current_librarian:
+        identity.provides.add(
```

Comment:
You can compute a list of organisation pids to add depending if it is a librarian or a patrons and add the needs at the end of the function.

Reply:
of course. I tried but readability is degraded.